### ISSUE TITLE
When app is run in elevated admin mode AlwaysOnTop won't work

### ISSUE THREAD

**Issue by stephen147** [jparnell8839/AlwaysOnTop#12]
After running an app in elevated admin mode it can't be placed on top.

**Comment by jparnell8839** [comment]
Interesting... This is probably because the UAC model will not allow the process handle to be manipulated... Have you tried running AlwaysOnTop as admin under UAC?

**Comment by jparnell8839** [comment]
Nope, that's not a valid solution... If you have hotkeys enabled, every key gets intercepted... I'll take a look at this

**Comment by stephen147** [comment]
I have my computer in UAC mode and generally don't run apps in an elevated state as it causes havoc with dragging and dropping. Hopefully, you can find a fix.

**Comment by lankymart** [comment]
Is this issue related to Keyboard issue? #15
